src/asset-swapper/utils/quote_info.ts

```diff
@@ -27,14 +27,16 @@ export function calculateQuoteInfo(params: {
     slippage: number;
 }): QuoteInfo {
     const { path, operation, assetFillAmount, gasPrice, gasSchedule, slippage } = params;
+    const { nativeOrders, bridgeOrders, twoHopOrders } = path.getOrdersByType();
+    const singleHopOrders = [...nativeOrders, ...bridgeOrders];
     // TODO: generalize calculateQuoteInfo to handle a mix multihop+multiplex.
 
     // NOTES: until multihop+multiplex is supported a path will have single hop order(s) xor a two hop order.
     if (path.hasTwoHop()) {
-        return calculateTwoHopQuoteInfo(path.getOrders(), operation, gasSchedule, slippage);
+        return calculateTwoHopQuoteInfo(twoHopOrders[0], operation, gasSchedule, slippage);
```

Comment:
are we guaranteed only 1 twoHopOrder? a little confused by the use of the 0th index